client/my-sites/backup/backup-contents-page/file-browser/file-info-card.tsx

```diff
@@ -302,72 +306,75 @@ function FileInfoCard( {
 
 	const FileDetail = ( { label, value }: { label: string; value: string | number } ) => {
 		return (
-			<div className="file-card__detail">
-				<Text weight={ 700 }>{ label } </Text>
+			<HStack className="file-card__detail" justify="flex-start" spacing={ 1 }>
+				<Text weight={ 700 }>{ label }</Text>
 				<Text>{ value }</Text>
-			</div>
+			</HStack>
 		);
 	};
 
+	const hasMeta = item.type === 'table' || size || modifiedTime || fileInfo?.hash;
 	return (
-		<Card isRounded={ false } variant="secondary" isBorderless>
-			<CardBody>
+		<Card isRounded={ false } isBorderless className="file-card">
```

Comment:
These changes are mostly structural changes to align with the new design proposal.